frontend/index.html

```diff
@@ -117,5 +100,27 @@
         })
         .forEach((link) => document.head.append(link));
     </script>
+
+    <script
+      type="text/javascript"
+      src="https://preview.vaadin.com/vaadincom/haas-service/haas-loader.js"
+      defer={true}
```

Comment:
Why the `{` `}` ?

Reply:
Oh, my bad. Copy-paste leftover from a React codebase 😁

Reply:
Removed